docs/current_docs/install.mdx

```diff
@@ -9,6 +9,8 @@ import { daggerVersion } from './partials/version';
 
 # Installation
 
+## Installation
```

Comment:
Seeing `Installation` twice looks awkward.

How about renaming this to `Release type`? We would then rename `Release version` to `Stable release` and `Development release`

Reply:
Done